TaskLayer/SearchTask/SearchTask.cs

```diff
@@ -203,10 +204,21 @@ protected override MyTaskResults RunSpecific(string OutputFolder, List<DbForTask
             Parallel.For(0, currentRawFileList.Count, parallelOptions, spectraFileIndex =>
             {
                 var origDataFile = currentRawFileList[spectraFileIndex];
+                var fileSetting = fileSettingsList[spectraFileIndex];
+                //set protease to fileSpecificToml if it exists (default if not)
+                Protease = fileSetting.Protease;
```

Comment:
Make sure to use the one specified in the search task if protease is not in the file-specific settings!